packages/contracts/contracts/QuadraticFundingCuratorContract.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+
+import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+
+//import "./IPayoutStrategy.sol";
+
+import "./mocks/MetaPtr.sol";
```

Comment:
This isn't a mock, it's a contract/interface that's part of the contract

Reply:
moved to interfaces.